docs/README.md

```diff
@@ -950,6 +950,9 @@ This action is usually used in combination with [reportportal-summarize](#report
 In particular, this prepares maven command line options for Report Portal integration, building the endpoint, authentication, launch key, description.
 Default context information is also added (launch attributes), unless the `auto-configure` input is set to `false`.
 
+By using the `rp-static-launch-name` flag, you can determine whether 'LAUNCH_NAME' should be static or unique for each execution. By default, it is set to `false`. 
```

Comment:
```suggestion
By using the `rp-static-launch-name` flag, you can determine whether the launch name in Report Portal should be static or unique for each execution. By default, it is set to `false`.
```

Make it clearer for the consumer, they shouldn't have to go through the bash script implementation to understand what a new flag does.